proton-j/src/main/java/org/apache/qpid/proton/engine/impl/EventImpl.java

```diff
@@ -68,12 +69,19 @@ void clear()
     }
 
     @Override
-    public Type getType()
+    public EventType getEventType()
     {
         return type;
     }
 
     @Override
+    public Type getType() {
+        if (type instanceof Type)
```

Comment:
> I'd be more inclined to drop the Reactor events from the Type given that there are currently no users of those events yet (its never been released)

If by drop you mean move reactor events to a future o.a.q.p.reactor.ReactorEvent.Type then
yes, that is exactly what I had in mind for a perfect world. It would be a code-massacre, tho. Deleting them outright would be a mistake, they are useful, and used ;)